frontend/packages/kitconcept-core/package.json

```diff
@@ -38,25 +38,24 @@
     "@mbarde/volto-image-crop-widget",
     "@plonegovbr/volto-social-media",
     "@plone-collective/volto-authomatic",
-    "volto-form-block",
     "@kitconcept/volto-light-theme"
   ],
   "dependencies": {
     "@plone/components": "workspace:*",
     "@plone-collective/volto-authomatic": "3.0.0-alpha.1",
-    "@kitconcept/volto-light-theme": "6.0.0-alpha.24",
+    "@kitconcept/volto-light-theme": "6.0.0",
     "@mbarde/volto-image-crop-widget": "^0.5.1",
-    "@plonegovbr/volto-social-media": "^2.0.0-alpha.5",
```

Comment:
VLT already puts it, let's have the deps in one place only, so we only have to update them once.